pkg/apis/uiplugin/v1alpha1/types.go

```diff
@@ -149,6 +149,8 @@ type LokiStackReference struct {
 
 // UIPluginSpec is the specification for desired state of UIPlugin.
 //
+// +kubebuilder:validation:XValidation:rule="self.type == 'TroubleshootingPanel' || !has(self.troubleshootingPanel)", message="TroubleshootingPanel type is required to use Troubleshooting Panel configuration"
+// +kubebuilder:validation:XValidation:rule="self.type == 'DistributedTracing' || !has(self.distributedTracing)", message="DistributedTracing type is required to use Distributed Tracing configuration"
```

Comment:
same here

Reply:
Updated the wording of the message of all the DistributedTracing validations